src/pages/IndividualReview.jsx

```diff
@@ -0,0 +1,34 @@
+import {useParams} from "react-router-dom";
+import { useState, useEffect } from "react";
+import ReviewCard from "../components/ReviewCard";
+import { useNavigate} from 'react-router-dom'
+
+export default function IndividualReview() {
+
+    const [review, setReview] = useState({})
+    const navigate = useNavigate();
+    const { review_id } = useParams();
+    const [isLoading, setIsLoading] = useState(true)
+
+
+        useEffect(() => {
+        setIsLoading(true)
+        fetch(`https://final-boardgame-api.herokuapp.com/api/reviews/${review_id}`)
+            .then((response) => response.json())
+            .then((data) => {
+                setReview(data.review);
+                setIsLoading(false)
+            })
+    }, [])
+    const ID = review_id
```

Comment:
you already have access to review_id so you don't need to assign it to a variable